railties/lib/rails/generators/app_base.rb

```diff
@@ -188,12 +187,12 @@ def database_gemfile_entry # :doc:
         return [] if options[:skip_active_record]
         gem_name, gem_version = gem_for_database
         GemfileEntry.version gem_name, gem_version,
-                            "Use #{options[:database]} as the database for Active Record"
+                            ["Use #{options[:database]} as the database.", "To change database, read https://guides.rubyonrails.org/configuring.html#configuring-a-database"]
```

Comment:
I'd rather drop "client" since I don't think it's true technically (you'd change the gem if you're using a different database, period). But I like the rest of the change.